src/vmm/src/vstate.rs

```diff
@@ -225,7 +228,7 @@ impl Vm {
                     guest_phys_addr: region.start_addr().raw_value() as u64,
                     memory_size: region.len() as u64,
                     userspace_addr: host_addr as u64,
-                    flags: 0,
+                    flags: KVM_MEM_LOG_DIRTY_PAGES,
```

Comment:
Could this change cause any performance degradation ? Do we have any data on this ?

Reply:
We don't have data (we don't even have a baseline to compare to), but it should introduce a performance penalty with all the new work the kernel has to do to account for dirty pages. I'm working on an additional enable/disable dirty logging mechanism so that we don't take the penalty if we don't care about the dirty bitmap.